snoop/defaultsettings.py

```diff
@@ -553,3 +563,12 @@ def bool_env(value):
 to hit the database, which is not available during that stage
 of the CI build.
 """
+
+SNOOP_WEBDAV_MOUNT_DIR = os.path.join(os.getenv('MNT', '/mnt'), 'snoop-webdav-mounts', 'collections')
```

Comment:
I think these `SNOOP_WEBDAV_CONF_DIR` `SNOOP_WEBDAV_MOUNT_DIR`  `SNOOP_WEBDAV_SECRETS_DIR` are left-overs / dead code. Let's remove them, since our webdavs are going to be mounted in the same place as s3 (in the custom temp dir), we don't want any reference to `/mnt` or defaults into `/opt/hoover/snoop`